database/102-up-timescale/20231025144021-spooler_add-amm-lp-rewards.sql

```diff
@@ -0,0 +1,29 @@
+-- migrate:up
+
+CREATE TABLE amm_positions (
+	tick_lower INT NOT NULL,
+	tick_upper INT NOT NULL,
+	position_id uint256 UNIQUE NOT NULL,
+
+	-- token id
+	pool VARCHAR NOT NULL,
+
+	liquidity uint256 NOT NULL
+);
+
+CREATE TABLE pending_lp_rewards (
```

Comment:
This should include a mention that it's used to spool pending lp rewards, that this is used for the cache!